ttnn/cpp/ttnn/operations/data_movement/clone/clone_pybind.cpp

```diff
@@ -32,6 +33,7 @@ void bind_clone_operation(py::module& module) {
             py::kw_only(),
             py::arg("dtype") = std::nullopt,
             py::arg("memory_config") = std::nullopt,
+            py::arg("compute_kernel_config") = std::nullopt,
```

Comment:
It feels a bit weird to have computer kernel config for the clone operation

Reply:
Actually, we have compute_kernel when dtype conversion is needed, so I added compute_kernel_config